python/pipeline/utils/filter_config.py

```diff
@@ -0,0 +1,62 @@
+import datajoint as dj
```

Comment:
OK, so I dropped `filter_config.py` altogether from the commit, but I added it to the `gitignore` so that it does not get updated until I include. Does that work?

For key_hash function, I think I saw that too but I cannot recall where... so I added it there... I probably copied Edgar's, which could be ur code.

I used hashed key such that when I later add more filter options, I can then simply access by hashed key, rather than individually calling its name (well even before, I should know which table it came from) with its corresponding hyper-parameters. ..

Reply:
still don't understand the reason for hashed keys